### PR TITLE
feat[alpha release v0 mainnet]: update send prompt functionality on contract and add cli support

### PR DESCRIPTION
**Pull request by jorgeantonio21**

### CODE REVIEW

**Review by Cifko — APPROVED** [review]

File: sui/cli/src/main.rs [hunk 1]
```diff
@@ -160,23 +160,126 @@ enum DbCmds {
...
-    /// Submits an example prompt to the network.
-    SubmitTellMeAJokePrompt {
+    /// Sends a text prompt to the network to a user specified output destination
+    SendPrompt {
         #[arg(short, long)]
         package: Option<String>,
         #[arg(short, long)]
         model: String,
-        #[arg(long, default_value_t = 1_000)]
+        #[arg(short, long)]
```
Comment (Cifko) [inline sui/cli/src/main.rs:169]: Too many conflicting `short`s in this pr. Just use long for everything in these commands.

File: sui/cli/src/main.rs [hunk 2]
```diff
@@ -160,23 +160,126 @@ enum DbCmds {
...
+        prompt: String,
+        #[arg(short, long)]
+        max_tokens: u64,
+        #[arg(short, long)]
+        temperature: u32,
```
Comment (Cifko) [inline sui/cli/src/main.rs:174]: `temperature` should be float, and the conversion should be in the code

**Review by Cifko — CHANGES_REQUESTED** [review]

File: sui/packages/atoma/sources/prompts.move [hunk 3]
```diff
@@ -83,30 +75,29 @@ module atoma::prompts {
...
-    entry fun generate_nft(
+    /// Submits a text prompt to Atoma network to generate a new image
+    entry fun send_image_generation_prompt(
         atoma: &mut AtomaDb,
         wallet: &mut Coin<TOMA>,
         model: ascii::String,
+        guidance_scale: u32,
+        prompt: vector<u8>,
+        uncond_prompt: vector<u8>,
+        height: u64,
```
[the remark this answers is not in the document]
Comment (jorgeantonio21) [reply sui/packages/atoma/sources/prompts.move:86]: I am following the convention in the contract method (alphabetic order)

File: sui/cli/src/main.rs (hunk 2 above)
Comment (jorgeantonio21) [reply sui/cli/src/main.rs]: Nice catch ! Corrected!

**Review by Cifko — APPROVED** [review]

**Pull request merged**